ballerina/Ballerina.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+org = "ballerina"
+name = "observe"
+version = "1.0.1"
+distribution = "slbeta3"
+export = ["observe", "observe.mockextension" ]
+
+[[platform.java11.dependency]]
+path = "../native/build/libs/observe-native-1.0.1-SNAPSHOT.jar"
+groupId = "ballerina"
+artifactId = "observe"
+
+[[platform.java11.dependency]]
+groupId = "io.opentelemetry"
+artifactId = "opentelemetry-sdk-trace"
+version = "1.0.0"
+
+[[platform.java11.dependency]]
+groupId = "io.opentelemetry"
+artifactId = "opentelemetry-sdk-testing"
+version = "1.0.0"
+
+[[platform.java11.dependency]]
+groupId = "io.opentelemetry"
+artifactId = "opentelemetry-sdk-common"
+version = "1.0.0"
+
+[[platform.java11.dependency]]
+groupId = "io.opentelemetry"
+artifactId = "opentelemetry-semconv"
+version = "1.0.0-alpha"
```

Comment:
The versions of the dependencies also need to be automatically replaced with placesholder committed. The versions should be only present in the gradle.properties.

Refer:
Placeholder: https://github.com/ballerina-platform/module-ballerinax-choreo/blob/main/choreo-extension-ballerina/Ballerina.toml#L65
Task: https://github.com/ballerina-platform/module-ballerinax-choreo/blob/main/choreo-extension-ballerina/build.gradle#L145-L173

Reply:
I am reading from https://github.com/ballerina-platform/module-ballerina-observe/pull/1/files#diff-54c2a1c9d62367d38139ae3d5d8200e06bc932ab7905f7e666d2c895382b2b81 file and replacing the versions as http